examples/master/index.js

```diff
@@ -1,7 +1,7 @@
 import {HotKeys, HotKeyMapMixin} from 'react-hotkeys';
 import React from 'react';
 import ReactDOM from 'react-dom';
-import rand from 'lodash/number/random';
+import rand from 'lodash/random';
```

Comment:
It's fine to use lodash in the example code, so I didn't remove it here. However this code appeared to be based on an older version of lodash, and so it was broken.